src/documentation/pages/components-wip/footer/examples/_external-brightspeed.vue

```diff
@@ -0,0 +1,236 @@
+              .chi-footer__search
+                .chi-form__item
+                  label.chi-label.-d--none(for="chi-footer-search-input") Search Lumen
+                  input.chi-input(type="text" placeholder="Search" id="chi-footer-search-input")
+            .chi-footer__links.chi-grid
+              .chi-col.-w--12.-w-md--6.-w-lg--3
+                ul.-text--bold
+                  li(v-for="(contactLink, index) in externalContents.contactLinks" :key="index")
+                    a(:href="contactLink.href" :target="contactLink.target" :class="contactLink.class") {{contactLink.title}}
+                .chi-footer__social
+                  a(v-for="(socialLink, index) in externalContents.socialLinks" :key="index" :href="socialLink.href" :aria-label="socialLink.ariaLabel" rel="noopener" target="_blank")
+                    i(:class="`chi-icon icon-logo-${socialLink.iconName} -md`" aria-hidden="true")
+              .chi-col.-w--12.-w-md--6.-w-lg--3
+                ul.-text--bold
+                  li(v-for="(aboutLink, index) in externalContents.aboutLinks" :key="index")
+                    a(:href="aboutLink.href" :target="aboutLink.target" :class="aboutLink.class") {{aboutLink.title}}
+              .chi-col.-w--12.-w-md--6.-w-lg--3
+                .chi-footer__links-title Solutions
+                ul
+                  li(v-for="(solutionLink, index) in externalContents.solutionLinks" :key="index")
+                    a(:href="solutionLink.href" :target="solutionLink.target" :class="solutionLink.class") {{solutionLink.title}}
+                .chi-footer__links-title Shop
+                ul
+                  li(v-for="(shopLink, index) in externalContents.shopLinks" :key="index")
+                    a(:href="shopLink.href" :target="shopLink.target" :class="shopLink.class") {{shopLink.title}}
+              .chi-col.-w--12.-w-md--6.-w-lg--3
+                .chi-footer__links-title Resources
+                ul
+                  li(v-for="(resourceLink, index) in externalContents.resourceLinks" :key="index")
+                    a(:href="resourceLink.href" :target="resourceLink.target" :class="resourceLink.class") {{resourceLink.title}}
+        .chi-footer__internal
+          .chi-footer__internal-content.-mw--1200
+            .chi-dropdown.chi-footer__language
+              a.chi-button.-icon.-flat.-light.-sm.chi-dropdown__trigger(id="language__dropdown_button" ref="language-dropdown-button" data-position="top-start" aria-label="Select your preferred language")
```

Comment:
Version dropdown is not initialized. In production, it is fully functional.

Reply:
I did not get this comment. I believe we only have dropdown for languages in footer.

Reply:
Version dropdown used in Footer does not work in this implementation. It is functional in production.

Reply:
I don't see any version dropdown used in footer. Do you mean the version dropdown in header? Just want to know the relativity of the version dropdown with footer component.